config/set/php73.php

```diff
@@ -37,7 +36,6 @@
     $rectorConfig->rules([
         StringifyStrNeedlesRector::class,
         RegexDashEscapeRector::class,
-        ContinueToBreakInSwitchRector::class,
```

Comment:
it exists already in php52 config set, I removed it to avoid duplicated error message.

```bash
cd e2e/with-php-level && composer install
with-php-level git:(experiment-use-composer-json-priority-fallback) ✗ php ../e2eTestRunner.php
    ---------- begin diff ----------
@@ @@
-[ERROR] Following rules are registered twice: Rector\Php52\Rector\Switch_\ContinueToBreakInSwitchRector
```